showcase-docs/getting-started.md

```diff
@@ -25,6 +25,17 @@ The easiest and fastest method for getting started with the Backstage Showcase a
    ```yaml
    # This is a GitHub App. You can find out how to generate this file, and more information
    # about setting up the GitHub integration here: https://backstage.io/docs/integrations/github/github-apps
+
+   proxy:
+     '/sonarqube':
+       target: ${SONARQUBE_URL}/api
+       allowedMethods: ['GET']
+       auth: '${SONARQUBE_TOKEN}:'
+
+   # comment this out if using sonarcloud
+   sonarQube:
+     baseUrl: ${SONARQUBE_URL}
```

Comment:
Do we need both the `target` and the `baseUrl`

Reply:
apparently yes: https://roadie.io/backstage/plugins/sonarqube/#:~:text=sonarQube%3A%0A%20%20baseUrl%3A%20https%3A//your.sonarqube.instance.com